packages/arb-token-bridge-ui/src/util/cctp/messengerTransmitterAbi.ts

```diff
@@ -0,0 +1,451 @@
+export const messengerTransmitterAbi = [
```

Comment:
Should be `MessageTransmitter`: https://developers.circle.com/stablecoin/docs/cctp-messagetransmitter

Reply:
all other vars named `messengerTransmitterXX` has to be renamed too